testcases/dynamic_client/dynamic_client.robot

```diff
@@ -0,0 +1,44 @@
+*** Settings ***
+Resource          ./dynamic_client_kw.robot
+
+*** Test Cases ***
+Dynamic client test case example
+    [Tags]    dynamic-client
+    ${resources}=    discover resources    default
+    Log To Console     ${resources}
+Dynamic client test case example 2
+    [Tags]     dynamic-client
+    ${conf}=     read conf     testcases/dynamic_client/resources/pod.yaml
+    create pod     ${conf}
+    sleep     15 seconds
+    ${pods}=     get specific pod      default      app=myapp
+    ${metadata}=     Get From Dictionary    ${conf}      metadata
+    ${patched_labels}=     Create Dictionary      app=myapp       tested=true
+    ${patched_metadata}=     Set To Dictionary     ${metadata}      labels      ${patched_labels}
+    ${patched_pod}=      Set To Dictionary     ${conf}      metadata      ${patched_metadata}
+    patch pod    ${patched_pod}
+    sleep     15 seconds
```

Comment:
Could we make this sleep smaller, or use sth like wait until keywords succeeds

Reply:
Agreed.